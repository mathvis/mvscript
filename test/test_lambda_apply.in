((a: int): {a})(6)
