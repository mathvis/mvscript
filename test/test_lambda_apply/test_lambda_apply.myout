Expr (LambdaApplication (LambdaFunc [(VarIdentifier "a",IntT)] (Just (Block (FunctionBlock VoidT) [Expr (VarIdentifier "a")]))) (Type (Int 6)))
