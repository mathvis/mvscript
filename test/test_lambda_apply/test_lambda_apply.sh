#!/bin/bash
helix test_lambda_apply.mvs test_lambda_apply.out --vsplit
