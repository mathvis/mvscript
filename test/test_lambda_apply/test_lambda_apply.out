Expr (LambdaApplication (LambdaFunc [(VarIdentifier "a",int)] (Just (Block (FunctionBlock void) [Expr (VarIdentifier "a")]))) (Type (Int 6)))
